eng/testing/linker/project.csproj.template

```diff
@@ -1,5 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
+  <PropertyGroup>
+    <!-- Needed for PublishTrimmed -->
+    <ToolsILLinkDir>{ToolsILLinkDir}</ToolsILLinkDir>
+  </PropertyGroup>
+
+  <Import Project="$(ToolsILLinkDir)build/Microsoft.NET.ILLink.Tasks.props" />
```

Comment:
Unfortunately this is still not the correct sequence point. The props file is expected to be imported before the project is evaluated, i.e. in the project's Directory.Build.props file. You can achieve that by setting the dependent `ToolsILLinkDir` property before an explicit `Sdk` import and then moving the new `Import` statement into the Directory.Build.props file.